tests/unit/sagemaker/inference_recommender/test_inference_recommender_mixin.py

```diff
@@ -175,6 +175,134 @@ def default_right_sized_model(model_package):
     )
 
 
+def test_right_size_default_with_model_name_successful(sagemaker_session, model):
```

Comment:
I think we already performed e2e test locally, please add corresponding integ tests for model name: https://github.com/aws/sagemaker-python-sdk/blob/master/tests/integ/test_inference_recommender.py

Reply:
Yeah will also add integ tests